lib/chartmogul.rb

```diff
@@ -87,5 +87,6 @@ def config
     config_accessor :account_token
     config_accessor :secret_key
     config_accessor :max_retries, MAX_RETRIES
+    config_accessor :api_base, API_BASE
```

Comment:
We need to invalidate the connection cache if any config like credentials, max retries or api_base changes.

Reply:
I think we already do it here https://github.com/chartmogul/chartmogul-ruby/blob/f99574a125ae0b6dca8148e992097aa84a3c9244/lib/chartmogul/config_attributes.rb#L17